plugins/chat/app/services/chat/create_message.rb

```diff
@@ -32,7 +32,8 @@ class CreateMessage
     policy :ensure_valid_thread_for_channel
     policy :ensure_thread_matches_parent
     model :uploads, optional: true
-    model :message, :instantiate_message
+    model :message_instance, :instantiate_message
```

Comment:
This is a temporary fix, but we had the message model and the message attribute in contract, both would get merged in context, resulting in one winning over the other. 

Longer term I would want to have the attribute called text or raw anyways, but I didn't want to do this in this PR.